views/js/appearance/index.php

```diff
@@ -1,35 +0,0 @@
-<?php
-/*
-* 2007-2019 PrestaShop
```

Comment:
Let's keep the index.php (see comment above)

Reply:
@matks I remove the index.php, because I remove the directory :)